src/ui/localization.ts

```diff
@@ -25,7 +25,7 @@ const localizeInit = () => {
                     'scene.export': 'Exportieren',
                     'scene.export.compressed-ply': 'Komprimiertes PLY',
                     'scene.export.splat': 'Splat',
-                    'scene.export.viewer': 'Viewer App',
+                    'scene.export.viewer': 'Viewer App...',
```

Comment:
What's with the `...`?

Reply:
shows a dialog will appear upon selection.